build_tools/python/e2e_test_framework/definitions/common_definitions.py

```diff
@@ -88,10 +89,19 @@ class InputDataFormat(Enum):
 class DeviceSpec(object):
   """Benchmark device specification."""
   id: str
+
   # Device vendor name. E.g., Pixel-6.
   vendor_name: str
+
+  # Host platform that runs the runtime. For CPU device type, this is usually
+  # same as where the benchmarks/tests run. For GPU device type, this is where
+  # the tools/runtimes run on. The tools/runtimes then deploy the workload on
+  # the target GPUs.
```

Comment:
I think we should be careful not to conflate tools and the runtime here. For instance, with Android GPU, we have an attached linux x86_64 device. That is where various scripts are executed. Then iree-benchmark-module (which I assume is what you were referring to with "tools") runs the Android arm_64 device and dispatches workloads to the Mali or whatever GPU. I guess this script doesn't need to know about the "meta-host" if you will because it's already running there, but let's make sure to keep the documentation clear.

Also, the module *is* running on the host platform also: it's just that the main workloads are dispatched to the device.


```suggestion
  # Host platform where the IREE runtime is running. For CPU device type, this
  # is usually the same as the device that workloads are dispatched to. With
  # a separate device, such as a GPU, however, the runtime and dispatched
  # workloads will run on different platforms.
```